src/objects/zcl_abapgit_object_ddlx.clas.abap

```diff
@@ -179,10 +182,12 @@ CLASS zcl_abapgit_object_ddlx IMPLEMENTATION.
           lr_data       TYPE REF TO data,
           lx_error      TYPE REF TO cx_root.
 
-    FIELD-SYMBOLS: <lg_data>    TYPE any,
-                   <lg_source>  TYPE data,
-                   <lg_version> TYPE data,
-                   <lg_package> TYPE data.
+    FIELD-SYMBOLS: <lg_data>       TYPE any,
+                   <lg_source>     TYPE data,
+                   <lg_version>    TYPE data,
+                   <lg_package>    TYPE data,
+                   <lg_changed_by> TYPE syuname,
+                   <lg_changed_at> TYPE xsddatetime_z.
```

Comment:
`TYPE any.`

Reply:
What's the advantage of type any. Isn't it good to type it here as precise as possible?

Reply:
released and exists on 702, https://abapedia.org/steampunk-2305-api/xsddatetime_z.dtel.html